docs/getting_started/user-guide.rst

```diff
@@ -41,6 +41,14 @@ spliced currently has the following predictors:
  - **smeagle**: is `another library being developed here <https://github.com/buildsi/Smeagle>`_ that is added but considered to be in experimental state. It requires `my branch of cle <https://github.com/vsoch/cle>`_.
 
 
+Debug information
```

Comment:
The seconds below also include information about the cache / container, so putting this header here could be misleading. I would either separate the two (so it's clear) or have this as part of one of the sections, or just below the list of predictors but not in a new section.

Reply:
I put it below the predictors and added links to it.